dom.bs

```diff
@@ -3821,7 +3815,7 @@ steps for each <a>exclusive <code>Text</code> node</a> <var>node</var> <a>descen
  <a>exclusive <code>Text</code> node</a>, if any.
 
  <li>Let <var>data</var> be the concatenation of the <a>data</a> of <var>node</var>'s
- <a>contiguous exclusive <code>Text</code> nodes</a> (excluding itself), in <a>tree order</a>.
```

Comment:
Hmm, the only other call site I see is "Remove node’s contiguous exclusive Text nodes (excluding itself), in tree order." That's also excluding node, it's removing exactly the same nodes whose data were concatenated earlier, anything else would be a bug.